asmcli/commands/install.sh

```diff
@@ -200,3 +177,49 @@ outro() {
 
   info "$(starline)"
 }
+
+configure_ca() {
+  local CA; CA="$(context_get-option "CA")"
+  case "${CA}" in
+    mesh_ca) configure_meshca;;
+    gcp_cas) configure_private_ca;;
+    citadel) configure_citadel;;
+  esac
+}
+
+configure_control_plane() {
+  if [[ "${MANAGED}" -eq 1 ]]; then
+    configure_managed_control_plane
+  fi
+}
+
+install_ca() {
+  local CUSTOM_CA; CUSTOM_CA="$(context_get-option "CUSTOM_CA")"
+  if [[ "${CUSTOM_CA}" -eq 1 ]]; then
+    local CA_CERT; CA_CERT="$(context_get-option "CA_CERT")"
+    local CA_KEY; CA_KEY="$(context_get-option "CA_KEY")"
+    local CA_ROOT; CA_ROOT="$(context_get-option "CA_ROOT")"
+    local CA_CHAIN; CA_CHAIN="$(context_get-option "CA_CHAIN")"
+
+    info "Installing certificates into the cluster..."
+    kubectl create secret generic cacerts -n istio-system \
+      --from-file="${CA_CERT}" \
+      --from-file="${CA_KEY}" \
+      --from-file="${CA_ROOT}" \
+      --from-file="${CA_CHAIN}"
+  fi
+}
```

Comment:
Everything looks good except this is Citadel specific, so it should co into a similar file as meshca.sh

Reply:
Moved to an install function in citadel.sh